### PR TITLE
Seperate licenses and minerals into their own categories

### PR DESCRIPTION
**Pull request by DarcyManoel**
**Feature:** This PR implements the feature request detailed and discussed in issue #{{insert number}}

## Feature Details
Created two new outfit categories and moved the relevant outfits into them. Minerals and Licenses, what goes in them is pretty self-explanatory.

## UI Screenshots
This looks bad:
Screen Shot 2022-09-13 at 6 01 00 pm
This is bearable:
Screen Shot 2022-09-13 at 6 00 00 pm

## Usage Examples
Reduces the clutter in the trash bin that the `special` category has become, feels a lot better having them separated into their own categories. More importantly, hiding the damn licenses from view if the player wants.

## Testing Done
Um, it works? It's just a category change, I went to the shops.

## Performance Impact
N/A

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -44,3 +44,5 @@ category "outfit"
 	"Hand to Hand"
 	"Unique"
 	"Special"
+	"Minerals"
+	"Licenses"
```
Comment (Anarchist2) [inline data/categories.txt:48]: Ordering by how important it is that the player keeps track of these outfits:

- "Unique": These outfits are basically regular outfits, but with limited availability.
- "Minerals": Outfits that are sold. Given that the player will more regularly interact with these outfits than Special outfits, I feel it belongs higher than it.
- "Special": Outfits that stay with the player. They don't really need to keep track of them.
- "Licenses": Glorified conditions.

```suggestion
	"Unique"
	"Minerals"
	"Special"
	"Licenses"
```

**Comment by Zitchas** [comment]
Looks good to me!

Is there any special code or anything that deals with stuff in the "Special" category? For instance, licenses are kind of outfits, but they stay with the player. Is that inherent to them being given as licenses, or was the category they were in have anything to do with it?

**Comment by DarcyManoel** [comment]
> Looks good to me!
> 
> 
> 
> Is there any special code or anything that deals with stuff in the "Special" category? For instance, licenses are kind of outfits, but they stay with the player. Is that inherent to them being given as licenses, or was the category they were in have anything to do with it?

Pretty sure the special category has nothing really special about it, just a regular category.

File: data/categories.txt [hunk 2]
```diff
@@ -43,4 +43,6 @@ category "outfit"
 	"Engines"
 	"Hand to Hand"
 	"Unique"
+	"Minerals"
 	"Special"
+	"Licenses"
```
Comment (RisingLeaf) [inline data/categories.txt:48]: `# Outfits of these types can be bought and sold via the outfitter.`
Not rally about your pr, but this comment is wrong about licenses, they cannot bet bought in the outfitter

**Pull request merged**